priv/templates/extended_bin

```diff
@@ -416,8 +428,10 @@ case "$1" in
         echo "Exec: $@" -- ${1+$ARGS}
         echo "Root: $ROOTDIR"
 
-        # Start the VM
+        # Start the VM, invoke pre/post hooks
+        [ "$PRE_START_HOOK" ] && . "$PRE_START_HOOK"
         exec "$@" -- ${1+$ARGS}
+        [ "$POST_START_HOOK" ] && . "$POST_START_HOOK"
```

Comment:
One problem with this (and affects all of these) is that the `post_start` event will either never be executed, or will only be executed when the release stops, which is obviously not desired for a `post_start` event handler. I think the best way to do this is by backgrounding the task, executing the `post_start` task, then waiting on the PID, instead of blocking on `exec`. I'm using this approach right now in some work I'm doing in Mix, because I wanted to trap exit signals and cleanly shut down the VM.